primitives/src/lib.rs

```diff
@@ -96,12 +96,10 @@ pub enum CurrencyId {
     Native = 5,
 }
 
-pub const TOKEN_DECIMAL: u128 = 1_000_000_000_000_000_000;
+pub const TOKEN_DECIMAL: u128 = 1_000_000_000_000;
```

Comment:
We should not use this anymore.
Also don't forget to change the faucet if decimals for currencies changes.